app/helpers/course/controller_helper.rb

```diff
@@ -19,7 +19,7 @@ def display_course_user(user)
   # @return [String] The user-visible string, including embedded HTML which will display the
   #   string within a link to bring to the User page.
   def link_to_course_user(user, options = {})
-    link_path = course_user_path(user.course, user)
+    link_path = user.approved? ? course_user_path(user.course, user) : ''
```

Comment:
This will still give a link, just with a blank href. Should we just get rid of the link altogether?